README.md

```diff
@@ -164,16 +161,16 @@ $ mvn diktat:check
 rule sets by editing the YAML file).
 
 ## Run with Gradle using diktat-gradle-plugin
-Requires a gradle version no lower than 5.3.
+Requires a gradle version no lower than 7.0
 
-This plugin is available since version 0.1.5. You can see how the plugin is configured in our examples: [build.gradle.kts](examples/gradle-kotlin-dsl/build.gradle.kts).
+You can see how the plugin is configured in our examples: [build.gradle.kts](examples/gradle-kotlin-dsl/build.gradle.kts).
 
 <details>
 <summary>Add this plugin to your `build.gradle.kts`:</summary>
 
 ```kotlin
 plugins {
-    id("com.saveourtool.diktat.diktat-gradle-plugin") version "1.2.5"
+    id("com.saveourtool.diktat.diktat-gradle-plugin") version "2.0.0"
```

Comment:
`com.saveourtool.diktat` ?

Reply:
if yes - can you please also update examples? 🙏

Reply:
updated